### ISSUE TITLE
Block Third Party Cookies -- Allow Users to Use a Global Exception List

### ISSUE THREAD

**Issue by weems** [brave/browser-laptop#1268]
- A recent user request to make platform.twitter.com unblocked so that they can view embedded tweets on a blog page made clear the need for a Global Exception List
- in EFF Privacy Badger which Blocks Third Party Cookies and Tracking, One Can Setup a Global Whitelist if so inclined: 

<img width="466" alt="EFF Privacy Badger Screenshot" src="https://cloud.githubusercontent.com/assets/470705/14336039/e37def64-fc25-11e5-8d00-8363a2cb6bb8.png">
- You can allow Third Party Cookies in Bravery -> Clicking 'Allow ads and Tracking'
- The Problem being that it is a binary, globally on or globally off. A user cannot just allow a specific site through (like platform.twitter.com for embedded tweets) on all of their sites.

**Comment by alexwykoff** [comment]
@bradleyrichter this is still an issue and worth a moment of consideration. I'm putting it on 1.2 for now.

**Comment by srirambv** [comment]
+1 from[ support ](https://linkbubble.zendesk.com/agent/tickets/7007)on not able to view embedded tweets

**Comment by davidcelis** [comment]
An exception would also be necessary to [allow Duo Security to remember devices](https://help.duo.com/s/article/ka070000000fzu6AAA/2189?language=en_US); I currently have to authenticate much more often on Brave than other browsers

**Comment by bsclifton** [comment]
Thanks for rounding these issues up, @luixxiul 😄

**Comment by diracdeltas** [comment]
i think this should be done as part of https://github.com/brave/browser-laptop/issues/1991 (i.e., allow adding global cookie exceptions in a cookie manager)

**Comment by eduardonunesp** [comment]
Nice catch ! I'd like to allow third party cookies from Jira only :P or they wouldn't open

**Comment by Shought** [comment]
+1 I also support this idea, https://community.brave.com/t/a-site-not-working-allowing-login-etc/8615

**Comment by bsclifton** [comment]
Closing in favor of https://github.com/brave/brave-browser/issues/778
